examples/systemd-non-root-enable.yaml

```diff
@@ -0,0 +1,11 @@
+volume: harpoon-volume # this needs to match the volume name passed to the podman run command
+targets:
+- name: harpoon
+  url: http://github.com/sallyom/harpoon
+  systemd:
+    targetPath: examples/systemd
+    root: false
+    nonRootHomeDir: "/home/somalley"
```

Comment:
* will update this with the nonroot-homedir of the ubuntu instance

Reply:
@cooktheryan what's the non-root homedir of the ubuntu instance in gh actions? :)